content/attacks/posts/2022-10-11-Mango-Markets.md

```diff
@@ -1,33 +1,36 @@
 ---
 date: 2022-10-11
+target-entities: Mango Markets
+entity-types: Protocol
```

Comment:
Rewrite as follows, and discover [new changes](https://github.com/1712n/dn-institute/tree/main#readme) with list of common examples
```code
entity-types:
  - DeFi
  - Exchange